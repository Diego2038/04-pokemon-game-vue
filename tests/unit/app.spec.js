import App from '@/App.vue'
import { shallowMount } from '@vue/test-utils'

const wrapper = shallowMount( App );

describe( 'Componente App.vue', () => {
  test('App debe hacer match son el snapshot', () => {
    // console.log( wrapper.html() );
    expect( wrapper.html() ).toMatchSnapshot();
  })
})