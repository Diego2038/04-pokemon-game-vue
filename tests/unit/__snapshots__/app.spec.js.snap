// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente App.vue App debe hacer match son el snapshot 1`] = `<pokemon-page-stub></pokemon-page-stub>`;
